unison-cli/src/Unison/Codebase/Editor/Output.hs

```diff
@@ -179,8 +179,8 @@ data Output
   | ListNames
       IsGlobal
       Int -- hq length to print References
-      [(Reference, Set (HQ'.HashQualified Name))] -- type match, type names
-      [(Referent, Set (HQ'.HashQualified Name))] -- term match, term names
+      [(Reference, [HQ'.HashQualified Name])] -- type match, type names
```

Comment:
Just curious, why this change?

Reply:
The `Set` was only made at the last instant to get the names in the order we want to display them. But Name's `Ord` instance no longer does that, it's just derived.